### PR TITLE
adds dot on the left side

### PR DESCRIPTION
**Pull request by adripedroza**

### CODE REVIEW

File: src/WeekStackedComponents/TimeCell.js [hunk 1]
```diff
@@ -69,10 +69,20 @@ const TimeCell = ({
...
+                <div
+                  className="rbc-current-time-indicator"
+                  style={{
+                    top: `calc(${(now.getMinutes() / 60) * 100}% - 1px)`,
+                  }}
+                />
+                <div
+                  className="rbc-current-time-dot"
+                  style={{
+                    top: `calc(${(now.getMinutes() / 60) * 100}% - 0.5rem)`,
```
Comment (jkhoang313) [inline src/WeekStackedComponents/TimeCell.js:82]: is there a reason we're using `rem` instead of px here?

**Review by adripedroza — COMMENTED** [review]
god i started this review but idk how i did that and idk how to close it

File: src/sass/time-grid.scss [hunk 2]
```diff
@@ -135,7 +135,19 @@
   z-index: 3;
   left: 0;
   right: 0;
-  height: 1px;
+  height: 2px;
```
Comment (jkhoang313) [inline src/sass/time-grid.scss:138]: are these changes missing in the `.less` files?

Comment (adripedroza) [reply src/sass/time-grid.scss:138]: (hurried typing noises) idk what ur talkin about....

File: src/sass/time-grid.scss [hunk 3]
```diff
@@ -135,7 +135,19 @@
...
+
+  background-color: $current-time-color;
```
Comment (jkhoang313) [inline src/sass/time-grid.scss:140]: these changes too? 😅

Comment (jkhoang313) [reply src/sass/time-grid.scss:140]: jk the diff are just in different places

**Review by jkhoang313 — APPROVED** [review]

File: src/sass/time-grid.scss (hunk 3 above)
Comment (adripedroza) [reply src/sass/time-grid.scss:140]: GOD YOU SCARED ME

**Pull request merged**